### DISCUSSION TITLE
iCloud Error with V2022.02.22.0 -

### DISCUSSION THREAD

**Discussion by torchd02ta** [tonesto7/fordpass-scriptable#76] [General]
Just got done doing a copy and paste with new JS. Went to launch the app and I got the following error:

Error on line 3195:81. iCloud is not supported. Make sure you are logged into iCloud in the device settings

**Comment by torchd02ta** [comment]
Still get the same error with v2022.02.27. Now it is on line 3225:81

**Reply by ztnichol** [reply]
I have same error, any solution?
